### DISCUSSION TITLE
UHC HIOS Plan ID

### DISCUSSION THREAD

**Discussion by rgiljohann** [CMSgov/price-transparency-guide#641] [Q&A]
Does anyone know how to find the UHC HIOS Plan/Product ID on their price transparency website https://transparency-in-coverage.uhc.com/?

I did not find a TOC file anywhere for all plans. All I could find was their plan names which do not necessarily match up to HIOS plan/product names. 

I am looking to find a specific plan and I was hoping to use the HIOS plan/product ID but could not find it on the UHC website. 

Any help would be appreciated. Thanks.

**Comment by bryanbriegel** [comment]
One thing we might recommend, when seeking HIOS plans/products - for any issuer, where it is not apparent - in this case as noted UHC, is to go to Healthcare.gov - https://www.healthcare.gov/health-and-dental-plan-datasets-for-researchers-and-issuers/ - and select the year of interest - then individual or group plan data.  Download the file, then filter on the issuer (UHC) of interest and the state of interest.  Substring the "Plan ID (Standard Component) left(10) to isolate the 10 byte Product ID.  Then, go to UHC and locate the .index file of interest - for example if you are interested in TX - 2022-12-01_UnitedHealthcare-of-Texas-Inc_index.json.  Open the index file and search for the 10 byte HIOS plan ID you isolated from the Healthcare.gov data.  This will take you to the list of MRFs associated with the product in that state.  In the case of UHC and TX - 40220TX008 for Individual plan rate file, and 40220TX007 for group MRF rate files.  This is a long walk around the block, but will allow you to locate the files of interest.
